### ISSUE TITLE
Debug mode is activated when using MongoDB logging

### ISSUE THREAD

**Issue by LrsK** [urbanadventurer/WhatWeb#300]
When using the flag `--log-mongo-database`, the debug log seems to be activated to stdout or stderr, even if the quiet flag is set.

Example:
`whatweb --log-mongo-database wwtest google.com`

Partial output:

```
D, [2020-03-19T09:27:47.888412 #45] DEBUG -- : MONGODB | EVENT: #<TopologyOpening topology=Unknown[]>
D, [2020-03-19T09:27:47.888533 #45] DEBUG -- : MONGODB | Topology type 'unknown' initializing.
D, [2020-03-19T09:27:47.888744 #45] DEBUG -- : MONGODB | EVENT: #<TopologyChanged prev=Unknown[] new=Unknown[0.0.0.0:27017]>
D, [2020-03-19T09:27:47.888801 #45] DEBUG -- : MONGODB | Topology type 'Unknown' changed to type 'Unknown'.
D, [2020-03-19T09:27:47.888960 #45] DEBUG -- : MONGODB | EVENT: #<ServerOpening address=0.0.0.0:27017 topology=Unknown[0.0.0.0:27017]>
D, [2020-03-19T09:27:47.889100 #45] DEBUG -- : MONGODB | Server 0.0.0.0:27017 initializing.
D, [2020-03-19T09:27:47.889307 #45] DEBUG -- : MONGODB | Waiting for up to 30.00 seconds for servers to be scanned: #<Cluster topology=Unknown[0.0.0.0:27017] servers=[#<Server address=0.0.0.0
```

**Comment by urbanadventurer** [comment]
Fixed in #304

**Comment by urbanadventurer** [comment]
Thanks @LrsK ! 🏆
